hnn_core/network.py

```diff
@@ -518,3 +518,32 @@ def plot_spikes(self, ax=None, show=True):
         if show:
             plt.show()
         return ax.get_figure()
+
+    def write_spikes(self, fname, trial_idx='all'):
+        """Write spike times to a file.
+
+        Parameters
+        ----------
+        fname : str
+            Full path to the output file (.txt)
```

Comment:
What motivated the use of a `.txt` file instead of say an `.npz` or `.mat` file?

Reply:
I just wanted to be consistent with the `dipole.write()` function and previous HNN I/O convention.

Reply:
Since testing new hnn-core output against old hnn output will be important for the foreseeable future, I plead that we leave files the same (.txt). We will have test functions to compare array values after loading from the file, but also being able to do a `diff` from the command line is useful.